alf/utils/video_recorder.py

```diff
@@ -299,6 +306,15 @@ def capture_frame(self,
                 else:
                     self._encode_image_frame(frame)
 
+            if self._append_blank_frames > 0 and is_last_step:
+                if self._blank_frame is None:
+                    self._blank_frame = np.zeros_like(self.last_frame)
+                for _ in range(self._append_blank_frames):
+                    if self.ansi_mode:
```

Comment:
This branch appears multiple times in this file. Can use a function to wrap it.

Reply:
done